### PR TITLE
feat(stack overview): introduce the new ux implementation for stack overview feature - Initial phase

### PR DESCRIPTION
**Pull request by arunkumars08**
Changes:
1. Changes the constants file to add more strings
2. Changes the template structure in overview.component.html to add more structure
3. Changes the style for supporting the structure.
4. Passes dynamic values from the component
5. Makes padding changes for recommender and stack components feature
6. Adds a common style in stack-details.component.scss
7. Adds a template in stack-details.component.html

Tracks #77.

### CODE REVIEW

**Comment by arunkumars08** [comment]
@humaton Please merge once reviewed.

**Comment by arunkumars08** [comment]
screen shot 2017-03-31 at 5 43 10 pm

Changes are this

File: src/app/stack/recommender/recommender.component.html [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
-                    <div class="list-group-item-text">
-                        {{recommendation.suggestion}} - {{recommendation.action}} {{recommendation.message}}
+            <div class="list-view-pf-main-info">
+                <div class="list-view-pf-left">
+                    <span class="pficon pficon-info" [ngClass]="{'dismissed': recommendation.isDismissed === true}"></span>
+                </div>
+                <div class="list-view-pf-body">
+                    <div class="list-view-pf-description" [ngClass]="{'dismissed': recommendation.isDismissed === true}">
+                        <!-- <div class="list-group-item-heading">
+                            {{recommendation.suggestion}}
```
Comment (invincibleJai) [inline src/app/stack/recommender/recommender.component.html]: Comments can be removed

File: src/app/stack/overview/overview.component.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            font-size: 12px;
+        }
+        .overview-code-metric-icon {
+            color: #3788c9;
+        }
+        .license-entries {
+            line-height: 25px;
+        }
+    }
+}
```
Comment (invincibleJai) [inline src/app/stack/overview/overview.component.scss]: New line missing at file end

File: src/app/stack/recommender/recommender.component.scss [hunk 3]
```diff
@@ -1,114 +1,118 @@
...
     }
-}
 
-.activate {
-    cursor: pointer;
-}
+    .activate {
+        cursor: pointer;
+    }
+}
```
Comment (invincibleJai) [inline src/app/stack/recommender/recommender.component.scss]: file should end with new line

**Pull request merged**